sw/device/lib/dif/dif_flash_ctrl.c

```diff
@@ -946,13 +934,10 @@ dif_result_t dif_flash_ctrl_lock_data_region_properties(
   }
 
   bool locked;
-  dif_result_t result =
-      dif_flash_ctrl_data_region_is_locked(handle, region, &locked);
-  if (result != kDifOk) {
-    return result;
-  }
+  DIF_RETURN_IF_ERROR(
+      dif_flash_ctrl_data_region_is_locked(handle, region, &locked));
   if (locked) {
-    return kDifLocked;
+    return kDifOk;
```

Comment:
Why are you returning `kDifOk` if the config is locked? (applies throughout)

Reply:
Note that this would be the case where the user is attempting to lock an already-locked lock. It's following guidance from https://github.com/lowRISC/opentitan/issues/13643#issuecomment-1184885229

Reply:
ah got it, pls disregard.